app/src/androidTest/java/com/github/se/polyfit/ui/screen/NutritionalInformationTest.kt

```diff
@@ -141,5 +147,52 @@ class NutritionalInfoTest : TestCase() {
     }
   }
 
+  @Test
+  fun verifySuccessfulNameChange() {
+    ComposeScreen.onComposeScreen<NutritionalInformationBody>(composeTestRule) {
+      composeTestRule.onNodeWithTag("MealName").assertIsDisplayed()
```

Comment:
i think its easier to read/maintain tests if you define the elements in a screen file rather than continual use of `onNodeWithTag`

Reply:
improved formatted